documentation/book/cluster-operator.adoc

```diff
@@ -87,6 +87,10 @@ The accepted JSON format is described in the <<jvm_json_config>> section.
 `kafka-rack`::
 A JSON string allowing the Kafka rack feature to be configured and used in rack-aware partition assignment for fault tolerance.
 The accepted JSON format is described in the <<kafka_rack>> section.
+`kafka-affinity`::
+A JSON or YAML string allowing control over how the Kafka pods are scheduled to nodes.
```

Comment:
Isn't it only YAML?

Reply:
http://yaml.org/spec/1.2/spec.html#id2759572

Reply:
Moving to CRDs I think that we should not encourage JSON usage anymore.
I think that JSON is the most used one and a lot of people will start to use it instead of YAML if we "say" that doint that is possible.

Reply:
Well the fact remains that YAML is a superset of JSON (i.e. valid JSON is valid YAML), so we can't prevent people using JSON if they *want* to. But I expect all the documentation, examples etc would be YAML, not least since that's the norm in k8s-land.